rewrite-xml/src/main/java/org/openrewrite/xml/internal/XmlNamespaceUtils.java

```diff
@@ -25,42 +24,38 @@
 import java.util.Optional;
 import java.util.stream.Collectors;
 
-public class XmlUtils {
+public class XmlNamespaceUtils {
 
-    private XmlUtils() {
+    private XmlNamespaceUtils() {
     }
 
     private static boolean isNamespaceAttribute(String name) {
         return name.startsWith("xmlns");
     }
 
-    @NonNull
     public static String getAttributeNameFor(String namespacePrefix) {
         return namespacePrefix.isEmpty() ? "xmlns" : "xmlns:" + namespacePrefix;
     }
 
-    @NonNull
     public static String extractNamespacePrefix(String name) {
-        if (!isNamespaceAttribute("xmlns")) {
+        if (!isNamespaceAttribute(name)) {
```

Comment:
@ammachado This change of mine seems to have broken `XPathMatcherTest`, but it didn't make sense to me to validate `"xmlns"` as String. Any insights here?